apps/client/src/mocks/index.ts

```diff
@@ -1,7 +1,7 @@
 import { worker } from "./browser";
 
-if (process.env.NODE_ENV === "development") {
-  worker.start();
+export async function initMocks() {
```

Comment:
사소하긴 하지만 컨벤션에 맞추는 게 좋아보일 것 같아요 👍
```suggestion
export const initMocks = async () => {
```

Reply:
엇 !! 습관이... 무섭네요 ㅠ 다른 프로젝트에서 화살표가 아닌 대체로 함수형을 사용해서 ㅠㅠ 포인트 감사합니다 !